.github/workflows/initial_CI.yml

```diff
@@ -0,0 +1,42 @@
+name: Django CI
+
+on:
+#   push:
+#     branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 20
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
```

Comment:
I don't think you really want to test all those python versions. 

You're writing an end application not a library that Python programmers would use.

Reply:
fixed